src/sentry/static/sentry/app/utils/withOrganization.tsx

```diff
@@ -29,4 +29,11 @@ const withOrganization = <P extends InjectedOrganizationProps>(
     }
   };
 
+export function isLightweightOrganization(
+  organization: Organization | LightWeightOrganization
+): organization is LightWeightOrganization {
+  const castedOrg = organization as Organization;
+  return !(castedOrg.projects && castedOrg.teams);
+}
```

Comment:
@leedongwei Is this reliable? Lightweight (currently) to the initial load of organization details, we defer loading projects/teams, but we re-attach it to the organization.

This is a temporary measure, but it's meant to maintain compatibility while we refactor components that depend on projects/teams on org.

Reply:
Are you concerned that the projects/teams might not be successfully fetched or attached to the organization?

Reply:
@leedongwei oops nevermind, this is wrong - `projects/teams` do not get re-attached to the org.